packages/server/graphql/types/IntegrationProviderAuthStrategyEnum.ts

```diff
@@ -7,7 +7,8 @@ const IntegrationProviderAuthStrategyEnum = new GraphQLEnumType({
     oauth1: {},
     oauth2: {},
     pat: {},
-    webhook: {}
+    webhook: {},
+    sharedSecret: {}
```

Comment:
@Dschoordsch `IntegrationProviderAuthStrategyEnum.graphql` is the single source of truth now, if we don't update that, it'll cause some bad stuff down the line

Reply:
Thanks for finding!